Stripe/STPShippingAddressViewController.m

```diff
@@ -137,8 +137,11 @@ - (void)createAndSetupViews {
                        forState:UIControlStateNormal];
     [headerView.button addTarget:self action:@selector(useBillingAddress:)
                 forControlEvents:UIControlEventTouchUpInside];
-    BOOL needsAddress = self.configuration.requiredShippingAddressFields & PKAddressFieldPostalAddress && !self.addressViewModel.isValid;
-    BOOL buttonVisible = (needsAddress && self.billingAddress != nil && !self.hasUsedBillingAddress);
+    PKAddressField requiredFields = self.configuration.requiredShippingAddressFields;
+    BOOL needsAddress = requiredFields & PKAddressFieldPostalAddress && !self.addressViewModel.isValid;
```

Comment:
For clarity I think we should put parens around the expression that is a bitwise AND here because it's easy to misinterpret what's happening.